configs/tiny/OLMo-300M.yaml

```diff
@@ -47,47 +47,53 @@ optimizer:
   name: adamw
   learning_rate: 6.0e-4
   weight_decay: 0.1
+  decay_norm_and_bias: true
+  decay_embeddings: false
   betas:
   - 0.9
   - 0.95
   metrics_log_interval: 10
 
 scheduler:
   name: cosine_with_warmup
-  t_warmup: 2000
+  units: tokens
+  t_warmup: 4194304000
+  t_max: 3e12
   alpha_f: 0.1
+  grad_clip_warmup_steps: null
+  grad_clip_warmup_factor: 5
+  warmup_min_lr: 0
 
 tokenizer:
-  identifier: tokenizers/allenai_eleuther-ai-gpt-neox-20b-pii-special.json
+  identifier: tokenizers/allenai_gpt-neox-olmo-dolma-v1_5.json
   truncate_direction: right
 
 save_folder: workspace/${run_name}  # doesn't matter since we'll upload to S3
-remote_save_folder: s3://allennlp-ananyaj/olmo-tiny/300M/${run_name}
+remote_save_folder: s3://ai2-llm/checkpoints/olmo-tiny/300M/${run_name}
 save_overwrite: false
-# Sharded checkpoints (best for restarts)
-save_interval: 5000
-save_num_checkpoints_to_keep: 3
+
 # Unsharded checkpoints (for ddp)
 save_interval_unsharded: 5000
-save_num_unsharded_checkpoints_to_keep: 3
+save_num_unsharded_checkpoints_to_keep: -1
```

Comment:
What does `-1` do?

Reply:
-1 is for keeping all checkpoints, but I'll double check